test/lint/lint-submodule.py

```
@@ -13,7 +13,7 @@
 
 def main():
     submodules_list = subprocess.check_output(['git', 'submodule', 'status', '--recursive'],
-                                                universal_newlines = True, encoding = 'utf8').rstrip('\n')
+                                                text = True, encoding = 'utf8').rstrip('\n')
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)